### PR TITLE
57064 custom trackable kids

### PR DESCRIPTION
**Pull request by jasonfine**

### CODE REVIEW

File: app/models/concerns/fae/trackable.rb [hunk 1]
```diff
@@ -106,5 +112,13 @@ def update_parent
       end
     end
 
+    def tracker_parent_attrs
```
Comment (jamesmk) [inline app/models/concerns/fae/trackable.rb]: @jasonfine I refactored this method so we don't need to pass around so many arguments

https://github.com/wearefine/fae/pull/144/commits/37a10ed6617ea0b4d28b1dae171d173ec6115f1e

**Comment by jamesmk** [comment]
@jasonfine I'm seeing failing tests on Travis and locally. Can you address?

thanks!

**Comment by jasonfine** [comment]
all set ya troll ;)

**Comment by jasonfine** [comment]
looking into travis issues

**Comment by jasonfine** [comment]
spec issues introduced with this branch are fixed, however

rspec ./spec/features/form_helpers/fae_input_spec.rb:23 # fae_input should display hint when clicked
rspec ./spec/features/form_helpers/fae_input_spec.rb:32 # fae_input should display dark hint when clicked

are both present in master prior to the changes made in this branch

**Comment by jamesmk** [comment]
@jasonfine Hah, sorry for the half finished idea. The tests passed so I was like 👍 :)

**Pull request merged**